### PR TITLE
[logging] Add projects test

### PR DESCRIPTION
**Pull request by mgirgisf**

### CODE REVIEW

**Review by vyzigold — APPROVED** [review]

File: roles/common/tasks/project_tests.yml
Comment (ayefimov-1) [inline roles/common/tasks/project_tests.yml:1]: These tests are common tests and should *not* refer to any specific functions or features.

Recommend name change:
- name: Verify projects "{{ item }}"

**Review by ayefimov-1 — CHANGES_REQUESTED** [review]
Some of the changes made defeat the purpose of having a "common" role for telemetry testing.  Common role plays, tasks and playbooks should be reusable and feature agnostic.

File: roles/common/tasks/main.yml
[the remark this answers is not in the document]
Comment (mgirgisf) [reply roles/common/tasks/main.yml:1]: The aim of using a main.yml is to provide a single entrypoint to the role, which will select the tests based on the defined variables. its the same way we use in pod-tests

**Review by ayefimov-1 — APPROVED** [review]
Thank you. Looks good.

**Pull request merged**